internal/store/cache/cache.go

```diff
@@ -0,0 +1,41 @@
+package cache
+
+import (
+	"errors"
+
+	"github.com/dgraph-io/ristretto"
+)
+
+var (
+	ErrParsing = errors.New("parsing error")
+)
+
+type Config struct {
+	NumCounters  int64 `yaml:"num_counters" mapstructure:"num_counters"  default:"10000000"`
+	MaxCost      int64 `yaml:"max_cost" mapstructure:"max_cost"  default:"1073741824"`
+	BufferItems  int64 `yaml:"buffer_items" mapstructure:"buffer_items"  default:"64"`
+	Metrics      bool  `yaml:"metrics" mapstructure:"metrics"  default:"true"`
+	TtlInSeconds int   `yaml:"ttl_in_seconds" mapstructure:"ttl_in_seconds"  default:"3600"`
+}
+
+type CacheRepository struct {
+	*ristretto.Cache
+	ttlInSeconds int
+}
+
+func NewCacheRepository(config Config) (CacheRepository, error) {
```

Comment:
Let's call it Cache only

```suggestion
func NewCache(config Config) (Cache, error) {
```